### PR TITLE
feat: update fundamental-styles to 0.10.0-rc.1

### PR DESCRIPTION
**Pull request by meganaconley**
### Description
- `List` updated to include partial navigation, full navigation, action items, selection, and bylines
- `Identifier` updated to include more states and renamed to `Avatar`
- `Time` complete refactor
- `SideNavigation` markup updated
- Miscellaneous CSS updates pulled in from `fundamental-styles`

See changelog for full updates: https://github.com/SAP/fundamental-styles/blob/master/CHANGELOG.md

fixes https://github.com/SAP/fundamental-react/issues/1029
fixes https://github.com/SAP/fundamental-react/issues/1027
fixes https://github.com/SAP/fundamental-react/issues/1028
fixes https://github.com/SAP/fundamental-react/issues/1038

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "@babel/runtime": "^7.8.0",
     "chain-function": "^1.0.1",
     "classnames": "^2.2.6",
-    "fundamental-styles": "0.8.2-rc.0",
+    "fundamental-styles": "0.10.0-rc.1",
```
Comment (jbadan) [inline package.json:50]: giphy-1

File: src/Avatar/__stories__/Avatar.stories.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+export default {
+    title: 'Component API/Avatar',
+    component: Avatar
+};
+
+export const primary = () => (<Avatar glyph='cart' size='m' />);
+
+export const sizes = () => (
+    <div className='fr-container'>
```
Comment (jbadan) [inline src/Avatar/__stories__/Avatar.stories.js:13]: This got changed out from underneath you - but this class is now `fddocs-container` - that will get you the spacing you want.

File: src/Image/Image.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { IMAGE_SIZES, IMAGE_TYPES } from '../utils/constants';
 import React, { useEffect } from 'react';
 
-/** Use an **Image** component to display images. */
+/** Use an **Image** component to display images. **DEPRECATED**. Use the Avatar component instead. */
```
Comment (jbadan) [inline src/Image/Image.js:6]: Should we just delete this as part of the breaking changes?

File: src/Avatar/Avatar.js [hunk 4]
```diff
@@ -68,18 +77,23 @@ Identifier.propTypes = {
...
-    'xl',
-    'xxl' */
-    size: PropTypes.oneOf(IDENTIFIER_SIZES)
+    'xl' */
+    size: PropTypes.oneOf(AVATAR_SIZES),
+    /** Apply tile background color */
+    tile: PropTypes.bool,
+    /** Use transparent background */
+    transparent: PropTypes.bool,
+    /** Include a zoom icon */
```
Comment (jbadan) [inline src/Avatar/Avatar.js:95]: Do you mind changing these new boolean's description to "Set to **true** to...." to follow the pattern in other components. I've been trying to think of a better way to get consistency in these so I'm all ears if anyone thinks of a better solution!

File: src/Time/Time.js [hunk 5]
```diff
@@ -104,86 +116,96 @@ class Time extends Component {
...
-                        name='hour'
-                        placeholder={'hh'}
-                        spinners={spinners}
-                        time={time}
-                        type={'Hours'}
-                        upButtonProps={hoursUpButtonProps}
-                        updateTime={this.updateTime}
-                        value={time.hour} />
+                    <div className='fd-time__col' onClick={() => this.updateActiveColumn('hour')}>
+                        <label className='fd-time__slider-label'>Hours</label>
```
Comment (jbadan) [inline src/Time/Time.js:120]: Should these labels be localized? Could we just add to `localizedText` and pull them out here?

File: src/Time/Time.js [hunk 6]
```diff
@@ -104,86 +116,96 @@ class Time extends Component {
...
+                            disabled={disabled}
+                            disableStyles={disableStyles}
+                            downButtonProps={hoursDownButtonProps}
+                            format12Hours={format12Hours}
+                            inputProps={hoursInputProps}
+                            localizedText={localizedText}
+                            max={max}
+                            name='hour'
+                            time={time}
+                            type={'Hours'}
```
Comment (jbadan) [inline src/Time/Time.js:133]: Could we combine name and type and just have one prop or is there another purpose for them? I know those pre-existed this pr so we can totally make it a followup.

File: src/TreeView/__stories__/__snapshots__/TreeView.visual.storyshot [hunk 7]
```diff
@@ -14,7 +14,7 @@ exports[`Storyshots Visual Tree View 1`] = `
         <li
           aria-expanded={false}
           className="fd-tree__item"
-          id="fd-222"
+          id="fd-274"
```
Comment (jbadan) [inline src/TreeView/__stories__/__snapshots__/TreeView.visual.storyshot:17]: Side note: I'm realizing this https://github.com/SAP/fundamental-react/blob/master/storybook-testing/storyshots.test.js#L11 was a bad idea. After we merge this pr I'll put in a pr to change this so they don't all get affected with other changes

File: src/Time/Time.js (hunk 5 above)
Comment (meganaconley) [reply src/Time/Time.js]: Thanks for catching that, I noticed the `TimeItem` was also missing localized text so I've updated them all.

File: src/Time/Time.js [hunk 8]
```diff
@@ -104,86 +116,92 @@ class Time extends Component {
...
+                            inputProps={meridiemInputProps}
+                            localizedText={{ buttonUp: localizedText.meridiemUp, buttonDown: localizedText.meridiemDown }}
+                            max={'1'}
+                            name='meridiem'
+                            time={this.state.time}
+                            upButtonProps={meridiemUpButtonProps}
+                            updateActiveColumn={this.updateActiveColumn}
+                            updateTime={this.updateTime}
+                            value={this.CLOCK[this.state.time.meridiem]} />
+                    </div>
```
Comment (prsdthkr) [inline src/Time/Time.js:204]: Not sure if its related to this change, but the AM/PM selection seems off:
2020-05-21 16-20-21 gifcask 2020-05-21 16_24_13

File: src/Avatar/__stories__/__snapshots__/Avatar.visual.storyshot [hunk 9]
```diff
@@ -0,0 +1,346 @@
...
+        role="presentation"
+      />
+      <span
+        className="fd-avatar fd-avatar--m sap-icon--money-bills fd-has-background-color-accent-9"
+        role="presentation"
+      />
+    </div>
+  </div>
+</div>
+`;
```
[the remark this answers is not in the document]
Comment (meganaconley) [reply src/Avatar/__stories__/__snapshots__/Avatar.visual.storyshot:346]: Why yes, yes it should

File: src/Time/Time.js (hunk 8 above)
Comment (meganaconley) [reply src/Time/Time.js:205]: Thanks for catching, fixed

**Pull request merged**